wofs/wofs_app.py

```diff
@@ -456,30 +467,45 @@ def estimate_job_size(num_tasks):
 @click.group(help='Datacube WOfS')
 @click.version_option(version=__version__)
 def cli():
+    """
+    Instantiate a click.group object
+    :return: None
+    """
     pass
 
 
 @cli.command(name='list', help='List installed WOfS config files')
 def list_configs():
-    for cfg in CONFIG_DIR.glob('*.yaml'):
+    """
+     List installed WOfS config files
+    :return: None
+    """
+    for cfg in (glob.glob(os.path.abspath(os.path.join(WOFS_CONFIG_DIR,
```

Comment:
Why not something closer to what was there before? Please use the `pathlib.Path` style instead of the `os.path` style. If `WOFS_CONFIG_DIR` is fixed up, this can simply be:
`for cfg in WOFS_CONFIG_DIR.glob('*.yaml'*):` again.

Reply:
Revert back to previous logic, however replaced `WOFS_CONFIG_DIR` with `ROOT_DIR.parents[2].glob('wofs/config/*.yaml')`.